misc/scripts/install-local.sh

```diff
@@ -785,6 +785,15 @@ if ! checks; then
     return 1
 fi
 
+# If priority exists and is required, and also that this package has not been installed before (first time)
+if [[ -n ${priority} && ${priority} == 'required' ]] && ! is_package_installed "${name}"; then
+    ask "This package has 'priority=required', meaning once this is installed, it should be assumed to be uninstallable. Do you want to continue?" Y
+    if ((answer != 0)); then
+        cleanup
+        exit 1
```

Comment:
this prompt doesnt work btw it just kills the entire script regardless of response

Reply:
Huh? How?